beck-online.js

```diff
@@ -36,53 +36,69 @@
 
 
 var mappingClassNameToItemType = {
-	'ZAUFSATZ' : 'journalArticle',
-	'ZRSPR' : 'case',//Rechtssprechung
-	'ZRSPRAKT' : 'case',
-	'BECKRS' : 'case',
-	'ZENTB' : 'journalArticle',//Entscheidungsbesprechung
-	'ZBUCHB' : 'journalArticle',//Buchbesprechung
-	'ZSONST' : 'journalArticle',//Sonstiges, z.B. Vorwort,
-	'LSK'	: 'journalArticle', // Artikel in Leitsatzkartei
-	'ZINHALTVERZ' : 'multiple',//Inhaltsverzeichnis
-	'KOMMENTAR' : 'encyclopediaArticle',
-	'ALTEVERSION' : 'encyclopediaArticle',
-	'ALTEVERSION KOMMENTAR' : 'encyclopediaArticle',
-	'HANDBUCH' : 'encyclopediaArticle',
-	'BUCH' : 'book',
+	ZAUFSATZ: 'journalArticle',
+	ZRSPR: 'case', // Rechtssprechung
+	ZRSPRAKT: 'case',
+	BECKRS: 'case',
+	ZENTB: 'journalArticle', // Entscheidungsbesprechung
+	ZBUCHB: 'journalArticle', // Buchbesprechung
+	ZSONST: 'journalArticle', // Sonstiges, z.B. Vorwort,
+	LSK: 'journalArticle', // Artikel in Leitsatzkartei
+	ZINHALTVERZ: 'multiple', // Inhaltsverzeichnis
+	KOMMENTAR: 'encyclopediaArticle',
+	ALTEVERSION: 'encyclopediaArticle',
+	'ALTEVERSION KOMMENTAR': 'encyclopediaArticle',
+	HANDBUCH: 'encyclopediaArticle',
+	BUCH: 'book',
 	// ? 'FESTSCHRIFT' : 'bookSection'
 };
 
 // build a regular expression for author cleanup in authorRemoveTitlesEtc()
-var authorTitlesEtc = ['\\/','Dr\\.', '\\b[ji]ur\\.','\\bh\\. c\\.','Prof\\.',
-		'Professor', '\\bwiss\\.', 'Mitarbeiter(?:in)?', 'RA,?', 'PD',
-		'FAArbR', 'Fachanwalt für Insolvenzrecht', 'Rechtsanw[aä]lt(?:e|in)?',
-		'Richter am (?:AG|LG|OLG|BGH)',	'\\bzur Fussnote',
-		'LL\\.\\s?M\\.(?: \\(UCLA\\))?', '^Von', "\\*"];
+var authorTitlesEtc = ['\\/',
+	'Dr\\.',
+	'\\b[ji]ur\\.',
+	'\\bh\\. c\\.',
+	'Prof\\.',
+	'Professor(?:in)?',
+	'\\bwiss\\.',
+	'Mitarbeiter(?:in)?',
+	'RA,?',
+	'PD',
+	'FAArbR',
+	'Fachanwalt für Insolvenzrecht',
+	'Rechtsanw[aä]lt(?:e|in)?',
+	'Richter am (?:AG|LG|OLG|BGH)',
+	'\\bzur Fussnote',
+	'LL\\.\\s?M\\.(?: \\(UCLA\\))?',
+	'^Von',
+	"\\*"];
 var authorRegEx = new RegExp(authorTitlesEtc.join('|'), 'g');
 
 
-function detectWeb(doc, url) {
```

Comment:
I changed this because of a linting issue and I have seen this solution in another PR. Okay?